src/app/item-review/dtr-api-types.ts

```diff
@@ -0,0 +1,71 @@
+/** Information about the reviewer. Consistent between D1 and D2. */
+export interface DtrReviewer {
```

Comment:
Why isn't this just `DestinyAccount`?

Reply:
It's supplied from (and supplied to) the DTR API, and it doesn't have all the properties that `DestinyAccount` has.